test/e2e/lifecycle.go

```diff
@@ -84,6 +85,39 @@ func TearDown(client *Client) {
 	}
 }
 
+func (c *Client) LogsFor(namespace, name string, gvr schema.GroupVersionResource) (string, error) {
+	// Get all pods in this namespace.
+	pods, err := c.Kube.Kube.CoreV1().Pods(namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return "", err
+	}
+
+	logs := "" //map[string]string{}
+
+	// Look for a pod with the name that was passed in inside the pod name.
+	var pod *corev1.Pod
+	for _, p := range pods.Items {
+		if strings.Contains(p.Name, name) {
+			pod = &p
+			break
+		}
+	}
+
+	// Did we find a match like the given name?
+	if pod == nil {
+		return "", fmt.Errorf(`pod for "%s/%s" [%s] not found`, namespace, name, gvr.String())
+	}
+
+	// Collect all the logs from all the containers for this pod.
+	if l, err := c.Kube.Kube.CoreV1().Pods(namespace).GetLogs(pod.Name, &corev1.PodLogOptions{}).DoRaw(); err != nil {
+		logs = err.Error()
```

Comment:
```suggestion
		return "", err
```

Reply:
meh, this is in e2e and still working with it.